src/create_embeddings.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
```

Comment:
if you want to use the file as the wrapper around the functions, then lets change the name to something like `embeddings_service` or `embeddings_manager`, then the import `from src.embeddings_service import create_embeddings` looks way better.

you could rename `create_embeddings` -> `create`, but I can see both options working fine

Reply:
I have kept `create_embeddings` because it's explicit and makes it clearer.